### DISCUSSION TITLE
Problem when i will create a new localization for a linkmenuitem

### DISCUSSION THREAD

**Discussion by ODA669** [OrchardCMS/OrchardCore#13556] [General]
when i will try to create a new localization for a linkmenuitem or any element to menu its show me  this **   The page could not be found.   **

![image](https://user-images.githubusercontent.com/60682935/231581285-55bc790a-d8ee-400b-852b-dc4fef2523ee.png)

![image](https://user-images.githubusercontent.com/60682935/231581339-51e297d2-b0d0-4da3-97f6-394567205d7e.png)

I haven't done any changes in the module localization in or menu in orchard i dont understand for what show me that!

**Comment by hishamco** [comment]
Please steps to reproduce

**Comment by ODA669** [comment]
i create content menu item  and i added the localization part  when i was enter in the content menu item and i want add the culture en otherlanguage , it showme  the page not found

**Reply by hishamco** [reply]
I will try to reproduce the issue on the latest bits ..

**Reply by hishamco** [reply]
Yep, there's an issue. could you please clarify what you want to do? If your plan is to create localizable content menu items, I think this SHOULD NOT work as you think because the menu is dynamic data. @sebastienros correct me if I'm wrong

**Reply by ODA669** [reply]
i want to create a submenu in content items  for this i need the a culture different
